frontend/src/js/controllers/teamsCtrl.js

```diff
@@ -323,20 +323,26 @@
             });
         };
 
-
         vm.inviteOthers = function(ev, participantTeamId) {
             ev.stopPropagation();
-            // Appending dialog to document.body to cover sidenav in docs app
-            var confirm = $mdDialog.prompt()
-                .title('Invite others to this team')
-                .textContent('Enter the email address of the person')
-                .placeholder('email')
-                .ariaLabel('')
-                .targetEvent(ev)
-                .ok('Send Invite')
-                .cancel('Cancel');
 
-            $mdDialog.show(confirm).then(function(result) {
+            $mdDialog.show({
+                  clickOutsideToClose: true,
+                  scope: $scope,
+                  preserveScope: true,
+                  templateUrl: 'dist/views/web/invite-template.html',
+                  controller: function ($scope, $mdDialog) {
+                     $scope.closeDialog = function() {
+                        $mdDialog.hide();
+                     };
+                     $scope.result = function() {
+                        $mdDialog.hide(this);
+                     };
+                  }
+            })
+
+
+            .then(function(result) {
                 var parameters = {};
                 parameters.url = 'participants/participant_team/' + participantTeamId + '/invite';
```

Comment:
@Obtin This API should not be called in case of clicking on `Cancel` button. Please fix that.